src/common.js

```diff
@@ -397,3 +410,17 @@ function _register(name, implementation, wrapOptions) {
   }
 }
 exports.register = _register;
+
+exports.getProcessEnv = function () {
+  var env = objectAssign({}, process.env);
+
+  if (config.nodepath) {
+    env[PATH] += (process.platform === 'win32' ? ';' : ':') + path.join(process.cwd(), 'node_modules', '.bin') + (process.platform === 'win32' ? ';' : ':');
+  }
+
+  return env;
+};
+
+exports.getPathFromEnv = function (env) {
```

Comment:
I am hearing get rid of this change and the above logic and just make it use `process.env.PATH`, correct? 
Will make the change then.